scripts/setup-centos9.sh

```diff
@@ -175,6 +176,17 @@ function install_duckdb {
   fi
 }
 
+function install_stemmer {
+  wget_and_untar https://snowballstem.org/dist/libstemmer_c-${STEMMER_VERSION}.tar.gz stemmer
+  (
+    cd stemmer
+    sed -i '/CPPFLAGS=-Iinclude/ s/$/ -fPIC/' Makefile
+    make clean && make "-j${NPROC}"
+    ${SUDO} cp libstemmer.a /usr/local/lib/
```

Comment:
Is it possible to use cmake_install function as it has support for INSTALL_PREFIX instead of hard coding path.

Reply:
@kgpai, it is not a cmake project. So we cannot use cmake_install.

Reply:
Thanks @PHILO-HE , Could you then atleast incorporate changes to use INSTALL_PREFIX rather than hardcoding it to /usr/local/lib.